lib/pbench/test/functional/agent/cli/commands/tools/test_clear_tools.py

```diff
@@ -362,54 +371,70 @@ def test_clear_tools_test86(tools_configuration, groups, remotes, tools):
     out, err, exitcode = pytest.helpers.capture(command)
     assert b"" == out
 
-    empty_group_err_msg = "Blank group name specified; skipping."
-    empty_remote_err_msg = "Blank remote host name specified; skipping."
-    empty_tool_err_msg = "Blank tool name specified; skipping."
+    empty_group_err_msg = (
+        "Error: Invalid value for '-g' / '--group' / '--groups': "
+        "Blank group name specified; terminating.\n"
+    )
```

Comment:
I'm thinking that the "terminating" appendix doesn't add anything here:  I think it's obvious from context that we're terminating, and the fact that we're terminating has nothing to do with the "Invalid value" (that is, it's the other way around -- it's because of the invalid value that we're terminating!  😀).